docs/guides/edgemesh_test_env_guide.md

```diff
@@ -65,7 +65,7 @@ $ iptables -t nat -A PORT-MAP ! -i docker0 -p tcp -m tcp --dport portIN -j DNAT
 ![model](../images/edgemesh/model.jpg)
```

Comment:
line#8 :
get cni plugin and these five version are ~sopported~supported.

Reply:
Also , we can introduce a heading like Prerequisites at the beginning of the document , which then mentions to install containerd , etc.

Reply:
fix typo and add Prerequisites at the beginning of the document